doc/api/stream.md

```diff
@@ -917,6 +917,13 @@ will be ignored.
 Implementors should not override this method, but instead implement
 [`readable._destroy()`][readable-_destroy].
 
+##### readable.readableEncoding
+<!-- YAML
+added: REPLACEME
+-->
+
+Getter for the property `encoding` of a given `Readable` stream.
```

Comment:
Where is this set? Maybe we could add a link to the encoding explanation in this file.